lib/arclight/custom_document.rb

```diff
@@ -21,6 +21,7 @@ class CustomDocument < SolrEad::Document
       t.function(path: 'archdesc/controlaccess/function/text()', index_as: %i[displayable facetable])
       t.occupation(path: 'archdesc/controlaccess/occupation/text()', index_as: %i[displayable facetable])
       t.places(path: 'archdesc/controlaccess/geogname/text()', index_as: %i[displayable facetable symbol])
+      t.places_coll(path: "archdesc[@level='collection']/controlaccess/geogname/text()", index_as: %i[displayable facetable symbol])
```

Comment:
Hi @mejackreed - Yes, this is being used in a facet. I can try removing displayable.

@anarchivist - The purpose of this PR is to fix the collection level facets to not include those lower level heading.